src/test/java/com/minecolonies/util/DistanceSquaredTest.java

```diff
@@ -0,0 +1,57 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.ColonyManager;
+import net.minecraft.util.BlockPos;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.util.Random;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.number.OrderingComparison.greaterThanOrEqualTo;
+
+@PrepareForTest({ColonyManager.class, LanguageHandler.class})
+@RunWith(PowerMockRunner.class)
+public class DistanceSquaredTest
+{
+    /**
+     * Random seed.
+     */
+    private static final Random random = new Random(1839045834);
+
+    /**
+     * Nunber of iterations in testCase.
+     */
+    private static final int NOTestCases = 10000;
+
+    @Test
+    public void test2DTwoPositions()
+    {
+        for (int i = 0; i < NOTestCases; i++)
```

Comment:
We should just test normal values and extremes, using 10,000 random values is a bad test.